docs/ingestfile.rst

```diff
@@ -43,7 +43,7 @@ Extending from the example
 
 The above example uses a local filesystem to reduce the dependency on more complex connections and external environments like AWS, Google Cloud or 
 whatever storage you may have available. This is to keep the example simple. In real world scenario's, you'd probably write a bunch of operators to
-operate this way. The more incremental your process is, the more 
+operate this way. The more incremental your process is, the more operators you should expect to have.
```

Comment:
Thanks for reviewing!  I think it's best to write this differently and not make references to incremental loading. 

Something like this:

In real world scenario's, you'd probably incorporate where you keep data at rest into your operators. 

(remove "the more incremental ... the more" .

Can you update the PR?  Then I'll merge it.